app/views/comments/edit.js.erb

```diff
@@ -0,0 +1,11 @@
+commentContentP = $("#comment-content-<%= @comment.id %>");
+commentContentP.html("<%= escape_javascript(render 'form', comment: @comment, display_comment_field: true )%>");
+
+commentSubmitButton = commentContentP.find(".btn")
+commentSubmitButton.after("<small>   <%= escape_javascript(link_to 'Cancel', '#') %></small>")
+
+cancelLink = commentSubmitButton.next().find("a");
+cancelLink.on('click', function(e) {
+  e.preventDefault();
+  commentContentP.html("<%= escape_javascript(@comment.content) %>");
+});
```

Comment:
Is re-registering the handler necessary, or can you use the `$(document).on('event', 'selector' function)` live handler syntax?